doc/man1/openssl-pkcs12.pod

```diff
@@ -76,22 +76,10 @@ by default.
 The filename to write certificates and private keys to, standard output by
 default.  They are all written in PEM format.
 
-=item B<-passin> I<arg>
-
-The PKCS#12 file (i.e. input file) password source. For more information about
-the format of B<arg>
-see L<openssl(1)/Pass phrase options>.
-
-=item B<-passout> I<arg>
-
-Pass phrase source to encrypt any outputted private keys with. For more
-information about the format of B<arg> see the B<PASS PHRASE ARGUMENTS> section
-in L<openssl(1)>.
-
 =item B<-password> I<arg>
 
-With -export, -password is equivalent to -passout.
-Otherwise, -password is equivalent to -passin.
+With B<-export>, B<-password> is equivalent to B<-passout>;
+otherwise it is equivalent to B<-passin>.
```

Comment:
Shouldn't the semicolon be a comma?  Or well, I guess the semi is ok if you want to be -- what did you call it -- pretentious? :wink:

Reply:
Pretentious, moi? :)  Just checking if you were awake.  Fixed.